src/browser/module/connection/Connection.js

```diff
@@ -35,12 +35,7 @@ export class Connection extends kintoneBaseJSSDK.Connection {
       return kintone.api(super.getUri(restAPIName), String(methodName).toUpperCase(), body).then((response) => {
         return response;
       }).catch(err => {
-        const error = {
-          response: {
-            data: err
-          }
-        };
-        throw error;
+        throw new kintoneBaseJSSDK.KintoneAPIException(err);
```

Comment:
The constructor of `KintoneAPIException` is implemented like this:
```javascript
constructor(message = '', errors = {}, ...args) {
    ///...
}
```
So I think the first argument should be a string. I'm not sure what is the expected result here but if you want to pass the `err` object, I think this line should be:
```javascript
throw new kintoneBaseJSSDK.KintoneAPIException('', err);
```